devtools/cli/distribution/jreleaser.yml

```diff
@@ -0,0 +1,47 @@
+project:
+  name: quarkus
+  description: CLI for Quarkus
+  longDescription: |
+    Create projects, manage extensions and perform build and development tasks for Quarkus,
+    the Kubernetes Native Java stack tailored for OpenJDK HotSpot and GraalVM.
+  website: https://quarkus.io
+  authors:
+    - Quarkus Community
+  license: Apache-2
+  java:
+    groupId: io.quarkus
+    artifactId: quarkus-cli
+    version : 11
+  tags:
+    - cli
+    - quarkus
+    - java
+
+release:
+  github:
+    owner: quarkusio
+    name: quarkus
```

Comment:
The name of the repository matches the project name by default. This could be omitted unless explicit values are desired.

Reply:
y.. repo is `quarkus`, resource within is `quarkus-cli` .. being specific doesn't hurt. ALSO, for awhile, my local setup (which doesn't use origin as a remote name) was pretty confused, so I was trying to see if I could make it shut up by specifying more values so it wouldn't need to rummage around in the git repo.. 🤪

Reply:
Ah yes, the default remote name thingy -> https://github.com/jreleaser/jreleaser/issues/248
I believe the tool getting confused was also recently fixed https://github.com/jreleaser/jreleaser/issues/232
FWIW being explicit is OK, til someone complains that there's too much configuration in the file (YAML no less!! (wink wink))

Reply:
The `github` releaser currently requires a username (besides an OAuth token). If undefined will use the name of the repository owner, in this case "quarkusio". I have filed and fixed https://github.com/jreleaser/jreleaser/issues/256 to get around this problem.
I'll find a way to remove the strict requirement of a valid username.